components/web/php/php-8_2/Makefile

```diff
@@ -223,20 +226,20 @@ REQUIRED_PACKAGES += web/server/apache-24
 # Auto-generated dependencies
 REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
-REQUIRED_PACKAGES += $(ICU_LIBRARY_PKG)
 REQUIRED_PACKAGES += $(PG_LIBRARY_PKG)
 REQUIRED_PACKAGES += SUNWcs
 REQUIRED_PACKAGES += compress/bzip2
 REQUIRED_PACKAGES += database/sqlite-3
 REQUIRED_PACKAGES += library/gd
 REQUIRED_PACKAGES += library/gmp
+REQUIRED_PACKAGES += library/icu-75
```

Comment:
How you managed to get this line here?  Is your oi-userland repo up to date?

Reply:
Yes, my oi-userland is up to date. I did nothing special. I guess it is because of REQUIRED_PACKAGES += $(ICU_LIBRARY_PKG). And this is resolved like this
```
shared-macros.mk:ICU_LIBRARY_PKG =               library/icu-$(ICU_VERSION)
shared-macros.mk:ICU_VERSION =                   75
```

Reply:
No. `REQUIRED_PACKAGES` does not work that way.  All `REQUIRED_PACKAGES` lines below the `# Auto-generated dependencies` line are always removed and generated from scratch when you run `env - gmake REQUIRED_PACKAGES`.

Since there is `REQUIRED_PACKAGES_SUBST +=      ICU_LIBRARY_PKG` in `shared-macros.mk` then all occurrences of `library/icu-75` should be replaced by `$(ICU_LIBRARY_PKG)` in the `REQUIRED_PACKAGES` list.  Hence with the up-to-date `oi-userland` it is not possible to get the `REQUIRED_PACKAGES += library/icu-75` line (unless there is some not-yet-known bug somewhere).